fabrictestbed/cli/cli.py

```diff
@@ -149,12 +166,10 @@ def query(ctx, cmhost: str, ochost: str, tokenlocation: str, projectid: str, sco
         if sliceid is None:
             status, response = slice_manager.slices(includes=includes)
         else:
-            slice_object = Slice()
-            slice_object.slice_id = sliceid
-            status, response = slice_manager.get_slice_topology(slice_object=slice_object)
+            status, response = slice_manager.slices(includes=includes,slice_id=sliceid)
 
         if status == Status.OK:
-            click.echo(response)
+            click.echo(json.dumps(list(map(lambda i: i.to_dict(), response)),indent=2))
```

Comment:
I noticed that MyPy complains about the usage of `map` here:

`error: Argument 2 to "map" has incompatible type "Union[Exception, List[Any]]"; expected "Iterable[Any]"`

Maybe it doesn't matter since it is in the "OK" branch of the status check.

Reply:
I think that if "status==Status.OK" then it will not return an Exception.